prefab/curl-ssl/gradle.properties

```diff
@@ -19,4 +19,8 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# Enable importing Prefab dependencies
 android.enablePrefab=true
+
```

Comment:
Looks like you accidentally added a few empty lines to the end.

Reply:
that is true: removed!